syntax = "proto2";
import "game_event.proto";
import "card_attributes.proto";

message Event_SetCardAttr {
    extend GameEvent {
        optional Event_SetCardAttr ext = 2014;
    }
    optional string zone_name = 1;
    optional sint32 card_id = 2;
    optional CardAttribute attribute = 3;
    optional string attr_value = 4;
}
